internals/generators/container/index.js

```diff
@@ -20,10 +20,10 @@ module.exports = {
     },
   }, {
     type: 'list',
-    name: 'component',
-    message: 'Select a base component:',
-    default: 'PureComponent',
-    choices: () => ['PureComponent', 'Component'],
+    name: 'type',
+    message: 'Select the base component type:',
+    default: 'Stateless Function',
+    choices: () => ['Stateless Function', 'PureComponent', 'Component'],
```

Comment:
@samit4me  [React](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) calls them `React.PureComponent` and `React.Component`, because both of them are classes.